Support `GOOGLE_OAUTH_ACCESS_TOKEN` for Google Cloud Platform (#1578)
getsops/sops: README.rst

    $ gcloud auth application-default login

Using oauth tokens you can authorize by doing this:

.. code:: sh

    $ export CLOUDSDK_AUTH_ACCESS_TOKEN=<your access token>

> If you are already logged in 

Reviewer: Nit:
```suggestion
Or if you are logged in you can authorize by generating an access token:
```

.. code:: sh

>     $ export CLOUDSDK_AUTH_ACCESS_TOKEN=$(gcloud auth print-access-token)

Reviewer 2: ```suggestion
    $ export CLOUDSDK_AUTH_ACCESS_TOKEN="$(gcloud auth print-access-token)"
```

> 

Reviewer 2: Shouldn't you mention that `CLOUDSDK_AUTH_ACCESS_TOKEN` can also point to a file?

Also, does the Google Cloud SDK also support this double usage of `CLOUDSDK_AUTH_ACCESS_TOKEN`? If not, why support it here (but don't document it)?

Author: Yes, of course, i agree - i will remove the possibility to add it as a reference to a file.

  
Encrypting using GCP KMS
~~~~~~~~~~~~~~~~~~~~~~~~
GCP KMS has support for authorization with the use of `Application Default Credentials

> <https://developers.google.com/identity/protocols/application-default-credentials>`_ and using an oauth2 token.

Reviewer 2: ```suggestion
<https://developers.google.com/identity/protocols/application-default-credentials>`_ and using an OAuth 2.0 token.
```

    $ gcloud auth application-default login

> Using oauth tokens you can authorize by doing this:

Reviewer 2: ```suggestion
Using OAauth tokens you can authorize by doing this:
```
